app/org/sagebionetworks/bridge/models/accounts/StudyParticipant.java

```diff
@@ -300,6 +332,18 @@ public Builder withEmail(String email) {
             this.email = email;
             return this;
         }
+        public Builder withPhone(Phone phone) {
+            this.phone = phone;
+            return this;
+        }
+        public Builder withEmailVerified(Boolean emailVerified) {
+            this.emailVerified = emailVerified;
+            return this;
+        }
```

Comment:
This is the inverse of the HibernateAccount/GenericAccount case. In this case, we're storing StudyParticipants in Redis, which have verified emails but don't have the emailVerified flag? What will happen in this case? Is there any way to set the flag appropriately when read from Redis?

The simplest thing to do would be to clear Redis when we push to Prod, to get rid of legacy sessions, but that has the unfortunate side effect of logging everyone out.